src/config.rs

```diff
@@ -546,6 +551,7 @@ impl Default for DefaultCfConfig {
             max_bytes_for_level_multiplier: 10,
             compaction_style: DBCompactionStyle::Level,
             disable_auto_compactions: false,
+            disable_write_stall: true,
```

Comment:
To my understanding they should be `false` by default and controlled by `flow_control`?

Reply:
you're right